src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -833,89 +888,204 @@ export class MgtTasks extends MgtBaseComponent {
-      : html`
-          <span class="TaskCheck TaskIcon Incomplete"></span>
-        `;
+      : null;
+
+    const taskCheck = html`
+      <span class=${classMap(taskCheckClasses)}>${taskCheckContent}</span>
+    `;
+
+    const groupTitle = this._currentGroup ? null : this.getPlanTitle(task.topParentId);
+    const folderTitle = this._currentFolder ? null : this.getFolderName(task.immediateParentId);
+
+    const context = { task: { ...task, groupTitle, folderTitle } };
+    const taskTemplate = this.renderTemplate('task', context, task.id);
+    if (taskTemplate) {
+      return taskTemplate;
+    }
+
+    let taskDetails = this.renderTemplate('task-details', context, `task-details-${task.id}`);
 
-    const taskDresser =
-      this.dataSource === 'todo' || !this.isDefault(this._currentTargetDresser)
+    if (!taskDetails) {
+      const group =
+        this.dataSource === TasksSource.todo || this._currentGroup
+          ? null
+          : html`
+              <span class="TaskDetail TaskAssignee">
+                ${this.renderPlannerIcon()}
+                <span>${this.getPlanTitle(task.topParentId)}</span>
+              </span>
+            `;
+
+      const folder = this._currentFolder
         ? null
         : html`
-            <span class="TaskDetail TaskAssignee">
-              ${this.renderPlannerIcon()}
-              <span>${this.getPlanTitle(task.topParentId)}</span>
+            <span class="TaskDetail TaskBucket">
+              ${this.renderBucketIcon()}
+              <span>${this.getFolderName(task.immediateParentId)}</span>
             </span>
           `;
 
-    const taskDrawer = !this.isDefault(this._currentTargetDrawer)
-      ? null
-      : html`
-          <span class="TaskDetail TaskBucket">
-            ${this.renderBucketIcon()}
-            <span>${this.getDrawerName(task.immediateParentId)}</span>
-          </span>
-        `;
-
-    const taskDue = !dueDate
-      ? null
-      : html`
-          <span class="TaskDetail TaskDue">
-            ${this.renderCalendarIcon()}
-            <span>${getShortDateString(dueDate)}</span>
-          </span>
-        `;
-
-    const taskPeople =
-      !people || people.length === 0
+      const taskDue = !dueDate
         ? null
         : html`
-            <span class="TaskDetail TaskPeople">
-              ${people.map(
-                id =>
-                  html`
-                    <mgt-person user-id="${id}"></mgt-person>
-                  `
-              )}
+            <span class="TaskDetail TaskDue">
+              ${this.renderCalendarIcon()}
+              <span>${getShortDateString(dueDate)}</span>
             </span>
           `;
 
-    const taskDelete = this.readOnly
+      let taskPeople = null;
+      let assignedPeopleHTML = null;
+
+      const assignedPeople = Object.keys(assignments).map(key => {
+        return key;
+      });
+
+      let peopleExist = null;
+      if (assignedPeople.length === 0) {
+        peopleExist = html`
```

Comment:
You don't have to do this check, the template will be used if there are no people anyway. If you still want to use a variable, rename it to noPeopleTemplate

Reply:
Removed the check but kept the variable (renamed to `noPeopleTemplate`) for consistency: 00897b2